data/generate.fsx

```diff
@@ -43,7 +43,11 @@ type Table =
     StaticRows: Field list list
   }
 
-let gRNG = System.Random(123) // Fixed seed because we want constant values
+// Fixed seed because we want constant values
+// mutable because we want to easily reset and generate identical tables in CSV, easily
+// TODO: make this more robust, since this makes things depend on the order of generations
+let rngSeed = 123
+let mutable gRNG = System.Random(rngSeed)
```

Comment:
No need for mutable if I'm reading this right. You can rebind a value at a later point with the same name.

Reply:
I'm wrong since it re-runs the functions with implicit closure.